### PR TITLE
Revive faq

### PR DESCRIPTION
**Pull request by dschwabeS11**
- Move FAQ back into docs
- Move Flavors back into Compute
- Move lbaas and port security into network (subfolders) 
- Rename (git mv) all docs files from default.*.md to docs.*.md to apply the docs template
- Fix reference links
- Fix lint erros
- Remove tags as they are of no use and break the docs browsing feature (big arrows on left and right side)

### CODE REVIEW

File: pages/02.Documentation/02.compute/docs.de.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+### How long does a migration take?
+
+A live migration takes usually about 500ms. In some situations migrations may take longer.
+
+### Why are instances disconnected while migrating?
+
+To transfer the active state of instances (incl. RAM/Memory) they need to be 'frozen' prior to the migration. During the transfer network packets can get lost. It depends on the operating system and application that is being used if connection can be reestablished.
+
+### Kann ich einer Compute Instanz eine feste interne IP zuweisen??
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/02.compute/docs.de.md]: doppeltes Fragezeichen

File: pages/02.Documentation/02.compute/docs.en.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+```plain
+  management_port:
+    type: OS::Neutron::Port
+    properties:
+      network_id: { get_resource: management_net }
+      fixed_ips:
+        - ip_address: 192.168.122.100
+```
+
+### My compute instance was created, but is not accessible via SSH/HTTP
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/02.compute/docs.en.md]: why is the "etc." gone?

Comment (dschwabeS11) [reply pages/02.Documentation/02.compute/docs.en.md]: Syntax error. not allowed to use in headings

File: pages/02.Documentation/03.block-storage/docs.de.md [hunk 3]
```diff
@@ -25,4 +24,21 @@ Um die Leistung der virtuellen Volumen optimal auszunutzen sind folgende Einstel
...
 mkfs.ext4 -q -L myVolumeLabel -E stride=2048,stripe_width=10240
-```
\ No newline at end of file
+```
+
+---
+
+## Fragen & Antworten
+
+### Ist es möglich ein Cinder Volume auf mehreren VMs gleichzeitig zu verwenden?
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/03.block-storage/docs.de.md]: Komma fehlt hinter "möglich"

File: pages/02.Documentation/03.block-storage/docs.en.md [hunk 4]
```diff
@@ -25,4 +24,21 @@ To boost the performance of the virtual volumes the following settings are advis
...
 mkfs.ext4 -q -L myVolumeLabel -E stride=2048,stripe_width=10240
-```
\ No newline at end of file
+```
+
+---
+
+## Questions & Answers
+
+### Can I use a block storage volume across multiple VMs?
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/03.block-storage/docs.en.md]: "simultaneously" missing at the end

File: pages/02.Documentation/03.block-storage/docs.en.md [hunk 5]
```diff
@@ -25,4 +24,21 @@ To boost the performance of the virtual volumes the following settings are advis
...
+
+Unfortunately this is not possible. Think of a block storage volume like a virtual hard disk - just as you cannot connect a hard disk to multiple computers at the same time, you cannnot connect a block storage volume to multiple compute instances at the same time.
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/03.block-storage/docs.en.md]: can't or cannot with two 'n'

File: pages/02.Documentation/04.local-storage/docs.de.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+### Wann ist das Hardware-Wartungsfenster für local SSD Storage Nodes?
+
+Um die Auswirkungen vorhersehbar zu gestalten, kündigen wir hiermit ein Wartungsfenster an:
+
+>Jede Woche in der Nacht von Mittwoch, 23 Uhr bis Donnerstag, 4 Uhr,
+
+werden wir ca. 25% unserer Local SSD Storage Server warten. Statistisch gesehen wird jede Instanz **einmal im Monant heruntergefahren**.
+
+### Wie läuft eine Hardware-Wartung für local SSD storage nodes ab?
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/04.local-storage/docs.de.md]: Schreibweise Nodes/nodes mit Zeile 65 vereinheitlichen

File: pages/02.Documentation/04.local-storage/docs.de.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+Betroffene Instanzen erhalten zunächst ein ACPI Shutdown Signal und eine Frist von einer Minute, um sich ordentlich herunterzufahren. Rechnen sie mit bis zu einer halben Stunde Ausfallzeit, bevor die Instanzen wieder hochgefahren werden. Es sind neben dem ACPI Shutdown Event **keine weiteren Ankündigungen** vorgesehen.
+
+### Wieviele Instanzen sind gleichzeitig von Hardware-Wartungen der local SSD storage nodes betroffen?
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/04.local-storage/docs.de.md]: dito

File: pages/02.Documentation/04.local-storage/docs.en.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+---
+
+## Questions & Answers
+
+### Where can I find a list of all the Local SSD storage instance types?
+
+We provide a [tabular overview of our flavor types](../02.compute/docs.en.md).
+
+### How do I use local ssd storage?
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/04.local-storage/docs.en.md]: SSD in upper case

File: pages/02.Documentation/04.local-storage/docs.en.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+
+## Overview
+
+SysEleven Stack offers two flavors of ephemeral storage. The default storage type is a distributed storage based on quobyte. It offers high performance, redundancy and scalability at the cost of latency. For some application workloads, where latency is a bottleneck, we offer local ssd storage.
+
+---
+
+## Questions & Answers
+
+### Where can I find a list of all the Local SSD storage instance types?
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/04.local-storage/docs.en.md]: "local" in lower case

File: pages/02.Documentation/04.local-storage/docs.en.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+### What happens in case of hardware failures?
+
+Local SSD storage instances cannot be moved between hypervisors. That means in case of a hardware failure, **data loss is inevitable**.
+You need to replace them with a new system from scratch and data restored from one of your backups or surviving members of the cluster.
+
+### What about hardware maintenance?
+
+Local SSD storage instances cannot be (live) migrated, we need to regularly reboot our compute nodes for maintenance and this **inevitably affects all virtual machines with local storage** hosted on them.
+
+### When is the local storage node maintenance window?
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/04.local-storage/docs.en.md]: "SSD" missing between "storage" and "node"

File: pages/02.Documentation/07.network/docs.de.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+SysEleven Stacks Networking Service basiert auf dem OpenStack Neutron Projekt.
+
+Damit kannst du Network-Connectivity-as-a-Service für andere Dienste im SysEleven Stack nutzen, wie zum Beispiel den Compute Service.
+Außerdem stellt es Benutzern eine API zur Verfügung um Netzwerke zu definieren und um bestehende anzupassen.
+
+Du hast zwei Möglichkeiten dein Netzwerk zu verwalten. Du kannst dafür die öffentliche OpenStack API benutzen, genau so wie das [Dashboard](https://dashboard.cloud.syseleven.net).
+
+---
+
+## Fragen & Antworten
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/07.network/docs.de.md]: Die Reihenfolge der Fragen hat sich gegenüber vorher umgedreht, ich fand es vorher einleuchtender.

File: pages/02.Documentation/07.network/docs.en.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
...
+## Overview
+
+The SysEleven Stack Networking Service is built on the OpenStack Neutron project.
+It enables Network-Connectivity-as-a-Service for other SysEleven Stack services, such as the Compute Service. It provides an API for users to define networks and the attachments into them.
+
+You can manage your network both via our public OpenStack API endpoints, as well as using the [Dashboard](https://dashboard.cloud.syseleven.net).
+
+---
+
+## Questions & Answers
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/07.network/docs.en.md]: the order has been reversed, I found it sound before

File: pages/02.Documentation/09.orchestration/docs.de.md [hunk 13]
```diff
@@ -581,4 +583,19 @@ outputs:
...
     description: Floating IP address of loadbalancer in public network
     value: { get_attr: [ loadbalancer_floating_ip, floating_ip_address ] }
-```
\ No newline at end of file
+```
+
+---
+
+## Fragen & Antworten
+
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/09.orchestration/docs.de.md]: Die Frage zu Ubuntu 15.x Default Routen fehlt

File: pages/02.Documentation/09.orchestration/docs.en.md [hunk 14]
```diff
@@ -479,3 +481,18 @@ outputs:
     description: Floating IP address of loadbalancer in public network
     value: { get_attr: [ loadbalancer_floating_ip, floating_ip_address ] }
 ```
+
+---
+
+## Questions & Answers
+
```
Comment (mbernhardtSys11) [inline pages/02.Documentation/09.orchestration/docs.en.md]: the question about ubuntu 15.x default routes is missing

File: pages/02.Documentation/09.orchestration/docs.de.md (hunk 13 above)
Comment (dschwabeS11) [reply pages/02.Documentation/09.orchestration/docs.de.md]: Die sollte raus weil zu alt und nicht mehr brauchbar.

File: pages/02.Documentation/09.orchestration/docs.en.md (hunk 14 above)
Comment (dschwabeS11) [reply pages/02.Documentation/09.orchestration/docs.en.md]: see above

**Pull request merged**